js/context.js

```diff
@@ -5,12 +5,11 @@
 'use strict';
 
 // Use JSON style for context
-/* eslint quotes: ["error", "double"] */
-/* eslint quote-props: ["error", "always"] */
+/* eslint quotes: ['error', 'double'] */
+/* eslint quote-props: ['error', 'always'] */
 /* eslint-disable max-len */
 
-module.exports =
-{
+module.exports = {
```

Comment:
The odd style here was intentional in order to make the entire JSON block easy to copy.  Is there a reason to change it?

Reply:
Oh, I did it because that is the style I am use to, but I can change it back.